pallets/baby-liminal/Cargo.toml

```diff
@@ -74,6 +75,7 @@ runtime-benchmarks = [
     "frame-support/runtime-benchmarks",
     "frame-system/runtime-benchmarks",
     "sp-runtime/runtime-benchmarks",
+    "liminal-ark-poseidon",
```

Comment:
/std missing?

Reply:
I meant the "std" feature above benchmarks feature,  I cannot comment on non-edited lines :/

Reply:
it's because it's optional and it is needed only for benchmarks; perfectly, `poseidon` should be benchmarked outside the pallet, but I don't know how to do it quickly :man_shrugging: we can think about it later, it's not the worst place for that imho

Reply:
no, no it's a perfect place